app/controllers/oligrapher_controller.rb

```diff
@@ -94,6 +94,27 @@ def find_nodes
     render json: entities
   end
 
+  def find_connections
+    return head :bad_request if params[:entity_id].blank?
+
+    @entity = Entity.unscoped.find(params[:entity_id])
```

Comment:
You don't need to set an instance variable here since json is being rendered from the controller. A local variable named `entity` will work perfectly fine.